project-management/backlog-refinement-meetings/03-31-2020.md

```diff
@@ -0,0 +1,84 @@
+## Fujitsu/Turing backlog refinement meeting 03-31-2020
+
+### Attendees: (name/GitHub Username)
+
+* Georgia Aitkenhead/GeorgiaHCA
+* Ellen Devereux/EllenDevereux
+* Joe Collins/JoeC7756
+* Conor Henderson
+* Harrison Cannon
+* Jonathan Kerfoot
+* Ryan Galley
+* Jake Wych
+* Emilia Driscoll 
+* Christopher Legg 
```

Comment:
Is everyone comfortable having their full names in the GitHub repository? Could replace with initials and/or GitHub usernames? @fjThomasStanley